### ISSUE TITLE
Janno-Column for ENA Accession IDs

### ISSUE THREAD

**Issue by stschiff** [poseidon-framework/poseidon-schema#30]
We will need a column in the Janno-Files for ENA-Accession numbers leading to alignment files. This will then enable - in principle - systematic reanalyses of individuals described in Poseidon packages and in the long run enable more uniform processing of genotypes described in package repositories.

**Comment by nevrome** [comment]
The ENA Accession numbers are apparently [well defined](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/accessions.html). Cool! 

But what if the raw data is published not in the ENA, but on an other platform? Probably we should add a column that strictly adheres to the ENA number definition, but then also a column for a different identifier without any limitations.

**Comment by stschiff** [comment]
I think the only other platform that comes to mind is the [short read archive](https://www.ncbi.nlm.nih.gov/sra) by the NCBI. With a little luck, they have IDs that start with a characteristic string, such as ENAXXX or SRAXXX... we need to find out. That would solve the issue, as then we would simply ask maintainers to put in those IDs.

**Comment by stschiff** [comment]
So, I've googled a bit, [here](https://ena-docs.readthedocs.io/en/latest/index.html) is some docs on ENA submissions. What I've learned so far is that we'd definitely need this field to be a list column again. The reason is that samples are typically associated with several ENA accession IDs. You have one for the study, one for the sample, one for the sequencing run(s)... so I think we shouldn't over-specify this at this point. 

For example, what if a single sample has been sequenced multiple times, but the dataset relies on only one specific run. Then the correct accession ID would be a RUN ID instead of a SAMPLE ID. On the other hand, sometimes we have multiple runs for the same sample that have been merged, in which case the Sample ID would be more appropriate. So I think overall we should allow here multiple accession IDs and not specify what precise type of ID we require.

This would also have the advantage that we don't have to go and search for thousands of samples, but can in some cases simply put the study accession ID for all samples of a study, which is often the only accession ID listed in the method sections of papers. I'll put this also in a PR
